internal/integration/telemetry_sdk_emitter.go

```diff
@@ -207,9 +207,11 @@ func (te *TelemetryEmitter) Emit(metrics []Metric) error {
 				metric.value.(float64),
 				now,
 			)
-			if ok {
-				te.harvester.RecordMetric(m)
+			if !ok {
+				logrus.Infof("Got invalid delta for %s=%v, skipping", metric.name, metric.value)
```

Comment:
Can we change this to be debug? I am only worried this can fload us depending on the # of times it happens.
Do we have any idea?